app/src/main/java/com/zeapo/pwdstore/git/GitConfigActivity.kt

```diff
@@ -55,8 +61,10 @@ class GitConfigActivity : BaseGitActivity() {
                     putString("git_config_user_email", email)
                     putString("git_config_user_name", name)
                 }
-                    PasswordRepository.setUserName(name)
-                    PasswordRepository.setUserEmail(email)
+                PasswordRepository.setUserName(name)
+                PasswordRepository.setUserEmail(email)
+                Snackbar.make(binding.root, getString(R.string.git_server_config_save_success), Snackbar.LENGTH_SHORT).show()
+                Handler().postDelayed(500) { finish() }
```

Comment:
Shouldn't it be a bit longer to allow reading the snackbar message ?

Reply:
It seems long enough in practice.